src/editorKit.ts

```diff
@@ -75,7 +77,7 @@ export default class EditorKitBase {
   private fileSafeClass?: any
   private fileSafeInstance?: any
 
-  private note: any
+  private currentNote?: ItemMessagePayload
```

Comment:
```suggestion
  private note?: ItemMessagePayload
```

Can we name this back to note so we have a slimmer changelog here 😄

Reply:
Unless you have a reason for currentNote..

Reply:
Yeah, no reason to keep `currentNote`